git-extra/git-sdk.sh

```diff
@@ -72,12 +72,27 @@ sdk () {
 		'') break;;
 		-*) sdk die "Unknown option: %s\n" "$1"; return 1;;
 		esac; do shift; done &&
-		case "$(uname -m)" in
-		i686) bitness=" 32-bit";;
-		x86_64) bitness=" 64-bit";;
-		*) bitness=;;
+		case "$MSYSTEM" in
+		MINGW32) shortcut_suffix=" 32-bit";;
+		MINGW64) shortcut_suffix=" 64-bit";;
+		CLANGARM64) shortcut_suffix=" ARM64";;
+		MSYS)
+			case "$(uname -m)" in
+			i686) shortcut_suffix=" 32-bit";;
+			x86_64)
+				if test -d /clangarm64
+				then
+					shortcut_suffix=" ARM64"
```

Comment:
Do we want to make this lowercase `arm64` as discussed [here](https://github.com/git-for-windows/build-extra/pull/452#discussion_r1053792396)?

Reply:
That's a really good question. I am quite at a loss what we should do here. Maybe you can come up with sensible guidelines when to use upper-case "ARM64" and when to use lower-case "arm64" and when to use "clangarm64" and when to use "aarch64"?

Reply:
Will create a discussion in `git-for-windows/git` for that later today 👍🏼

Reply:
Slightly delayed, but here's the discussion: https://github.com/git-for-windows/git/discussions/4203

Reply:
> That's a really good question. I am quite at a loss what we should do here. Maybe you can come up with sensible guidelines when to use upper-case "ARM64" and when to use lower-case "arm64" and when to use "clangarm64" and when to use "aarch64"?

I just added some guidelines to the [architecture discussion](https://github.com/git-for-windows/git/discussions/4203). It basically means that the artifact suffix becomes `arm64` (lowercase). How does that sound? 😊

Reply:
> It basically means that the artifact suffix becomes `arm64` (lowercase). How does that sound? 😊

Sounds good! I did exactly that and force-pushed.